app/controllers/facility_reservations_controller.rb

```diff
@@ -43,42 +41,25 @@ def update
     @order_detail = @order.order_details.find(params[:order_detail_id])
     @reservation  = @order_detail.reservation
     @instrument   = @order_detail.product
-    if @reservation != Reservation.find(params[:id]) || @reservation.locked?
-      raise ActiveRecord::RecordNotFound
-    end
+    raise ActiveRecord::RecordNotFound if @reservation != Reservation.find(params[:id]) || @reservation.locked?
     set_windows
 
-    @reservation.assign_times_from_params(params[:reservation])
+    @reservation.assign_times_from_params(reservation_params)
 
     additional_notice = ""
 
-    if @order_detail.price_policy
-      update_prices
-
-      if @order_detail.actual_cost_changed? || @order_detail.actual_subsidy_changed?
-        additional_notice = "  Order detail actual cost has been updated as well."
-      end
-    else
-      # We're updating a reservation before it's been completed
-      if reserve_changed?
-        @order_detail.assign_estimated_price
-        additional_notice = " Estimated cost has been updated as well."
-      end
+    # We're updating a reservation before it's been completed
+    if reserve_changed?
```

Comment:
Where did we change this? I feel like I remember it and it's probably a good change, but it doesn't feel like it should be part of the strong params changes.

Reply:
It's this commit.

https://github.com/tablexi/nucore-open/pull/1419/commits/4d3f1c19cb1b9a5ad516b0c61e70ab9d0e692ad0

Do you feel strongly enough that it's worth separating out the tech debt cleanup part from the strong params change? Strong params is itself a bit of a tech debt cleanup task--I feel like I'm okay with this being here but can pull it out if it really bothers you.

Reply:
I don't feel strongly enough to be worth the effort.